app/components/Search/SearchResults.js

```diff
@@ -33,6 +34,13 @@ const SearchResultItem = ({
           <Icon className={styles.searchResultItemIcon} name={result.icon} />
         )}
       {result.label}
+      {result.date && (
+        <Time
+          time={Date.parse(result.date)}
+          wordsAgo
+          className={styles.resultDate}
```

Comment:
If we add other information in this spot, it would be better with a more generic name. If we only use it for dates/time, this is nice! :smile: